src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -234,55 +237,75 @@ private async Task<(IWebDriver browser, ILogs log)> CreateSauceBrowserAsync(stri
                 name = $"{name} - {context}";
             }
 
-            var capabilities = new DesiredCapabilities();
+            DriverOptions options;
+
+            switch (sauce.BrowserName.ToLower())
+            {
+                case "chrome":
+                    options = new ChromeOptions();
+                    break;
+                case "safari":
+                    options = new SafariOptions();
+                    break;
+                case "internet explorer":
+                    options = new InternetExplorerOptions();
+                    break;
+                case "microsoftedge":
+                    options = new EdgeOptions();
+                    break;
+                default:
+                    throw new InvalidOperationException($"Browser name {sauce.BrowserName} not recognized");
+            }
```

Comment:
Why do we need this? We ended up not going this route @ajaybhargavb isn't that the case?

We only need chrome I believe

Reply:
We only need Chrome for local and CI tests. But we use other browsers when running in SauceLabs. We need to pass in the correct options to SauceLabs in that case. `DesiredCapabilities` used to be enough but that is now obsoleted in the newer version.

Reply:
@ajaybhargavb but do we actually run in SauceLabs? I thought we discarded that idea. That's why I'm asking.

Reply:
We still want to. What we discarded was the idea to do real device testing on SauceLabs.